assets/javascript/app/dcmaw-async.js

```diff
@@ -0,0 +1,22 @@
+// The generated function is for use with the frontend-ui spinner component
+function generatePollApiFunction(url) {
+  return async function pollApiFunction(abortSignal) {
+    return fetch(url, { signal: abortSignal })
+      .then((response) => response.json())
+      .then((data) => {
+        if (data.status === "COMPLETED" || data.status === "INTERVENTION") {
```

Comment:
Is `INTERVENTION` an actual status? only it's not defined here: https://github.com/govuk-one-login/ipv-core-front/blob/fe400527e07ed4b005e410df050b4b40c90a9708/src/app/vc-receipt-status/middleware.ts#L9